### ISSUE TITLE
Package on PyPI

### ISSUE THREAD

**Issue by xiongchiamiov** [thomaspurchas/tarfile-Progress-Reporter#2]
[Packaging up the project](https://packaging.python.org/distributing/) would allow it be a simple `pip install` away for others, and allows them to keep up to date much easier with upstream changes than if they vendorize it.

**Comment by iamtalhaasghar** [comment]
Yup, this python library definitely needs to be packaged and shipped on PyPI.
